### PR TITLE
Move masonry grid to AdobeUi module

### PR DESCRIPTION
**Pull request by engcom-Golf**
### Description (*)
Move masonry grid to AdobeUi module (prepare for moving to magento2)

Moved the masonry grid to AdobeUI module:

- Images grid
- Image preview (only next, previous and close controls)

### Fixed Issues (if relevant)
1. magento/adobe-stock-integration#314: Move masonry grid to AdobeUi module (prepare for moving to magento2)

### Manual testing scenarios (*)
1. Configure Adobe stock
2. Search images.

### CODE REVIEW

File: AdobeUi/README.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+# Magento_AdobeStockImage Module
```
Comment (sivaschenko) [inline AdobeUi/README.md]: Please update readme

File: AdobeUi/Ui/Component/Listing/Columns/ImagePreview.php [hunk 2]
```diff
@@ -6,7 +6,7 @@
 
 declare(strict_types=1);
 
-namespace Magento\AdobeStockImageAdminUi\Ui\Component\Listing\Columns;
+namespace Magento\AdobeUi\Ui\Component\Listing\Columns;
```
Comment (sivaschenko) [inline AdobeUi/Ui/Component/Listing/Columns/ImagePreview.php]: Only image preview with navigation (next/forward/close) should be moved to AdobeUi. Displaying image attributes and buttons should still be part of AdobeStockImageAdminUi functionality (extending AdobeUi)

File: AdobeUi/composer.json [hunk 3]
```diff
@@ -0,0 +1,23 @@
+{
+    "name": "magento/module-ui",
```
Comment (sivaschenko) [inline AdobeUi/composer.json]:
```suggestion
    "name": "magento/module-adobe-ui",
```

File: AdobeUi/composer.json [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+    "description": "N/A",
+    "require": {
+        "php": "~7.1.3||~7.2.0",
+        "magento/framework": "*",
+        "magento/module-adobe-stock-asset-api": "*",
```
Comment (sivaschenko) [inline AdobeUi/composer.json]: AdobeUi module is an extraction of masonry grid and overlay that will be delivered to Magento core. It should not depend on other adobestock modules

**Review by sivaschenko — CHANGES_REQUESTED** [review]
Thanks for the pull request @engcom-Golf @Nazar65 ! Please review my comments

**Comment by engcom-Golf** [comment]
Hi @sivaschenko I'm updated code by your review notes.

File: AdobeUi/README.md (hunk 1 above)
Comment (engcom-Golf) [reply AdobeUi/README.md]: ok

**Review by sivaschenko — APPROVED** [review]

**Review by sivaschenko — APPROVED** [review]

**Pull request merged**